docs/content/docs/connectors/table/pulsar.md

```diff
@@ -413,5 +415,60 @@ CREATE TABLE users
+
+For detailed differences and recommendations, please see the following sections.
+
+### Components
+
+| Repository              | Pulsar DataStream Source | Pulsar DataStream Sink | Pulsar SQL Source | Pulsar SQL Sink |
+|-------------------------|--------------------------|------------------------|-------------------|-----------------|
+| apache/flink 1.14       | Yes                      |                        |                   |                 |
+| apache/flink 1.15       | Yes                      | Yes                    |                   |                 |
+| streamnative/flink 1.15 | Yes                      | Yes                    | Yes               | Yes             |
+
+### Release process, version and documentation
+
+| Repository        | Release                                                  | Versioning                                                       | DataStream Connector Documentation                                                                                              | SQL Connector Documentation |
+|-------------------|----------------------------------------------------------|------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| apache/flink      | Managed by Flink community, follow Flink release process | Standard Semantic Versioning, like 1.15.0                        | Maintained on [Official Flink Website](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/) | Not available yet           |
+| streamnative/flink | Managed by StreamNative team, follow monthly release.    | Use an extra digit on top of the Flink version, such as 1.15.0.1 | Not available yet, but similar to the [Official Flink Website](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/)                                                                  | Maintained in current doc   |
+
+### Issue and Feature tracking
+
+- For Pulsar DataStream connectors, please use [Flink JIRA](https://issues.apache.org/jira/projects/FLINK/issues) to submit bug reports and feature requests.
+- For Pulsar SQL connectors, since it's not merged back to official Flink repository yet, please use the [Issue List](https://github.com/streamnative/flink/issues) in streamnative/flink to submit bug reports and feature requests. Once the SQL Connector is merged back to the official repository, we will use the Flink JIRA as well.
```

Comment:
```suggestion
- For the Pulsar SQL connector, since it's not merged back to the official Flink repository yet, create an [issue](https://github.com/streamnative/flink/issues) in the `streamnative/flink` repository to submit bug reports and feature requests. Once the SQL Connector is merged back to the official Flink repository, use the Flink JIRA as well.
```